src/main/java/htsjdk/samtools/reference/IndexedFastaSequenceFile.java

```diff
@@ -111,6 +114,18 @@ public IndexedFastaSequenceFile(final Path path) throws FileNotFoundException {
         this(path, new FastaSequenceIndex((findRequiredFastaIndexFile(path))));
     }
 
+    /**
+     * Initialise the given indexed fasta sequence file stream.
+     * @param in The input stream to read the fasta file from.
+     * @param indexIn The input stream to read the fasta file index from.
+     */
+    public IndexedFastaSequenceFile(final SeekableStream in, final InputStream indexIn) {
```

Comment:
Would it make more sense to have this take SeekableStream and a preconstructed `FastSequenceIndex` instead of two streams?

Reply:
Yes. Done.